scipy/stats/tests/test_distributions.py

```diff
@@ -198,6 +198,28 @@ def test_vonmises_expect(self):
         assert_allclose(np.angle(res), loc % (2*np.pi))
         assert np.issubdtype(res.dtype, np.complexfloating)
 
+    @pytest.mark.parametrize("rvs_loc", [0, 2])
+    @pytest.mark.parametrize("rvs_scale", [1, 100])
```

Comment:
```suggestion
    @pytest.mark.parametrize("rvs_shape", [1, 100])
```
I have some other comments, so no need to fix this yet.

`stats.vonmises.nnlf` should be doing the same thing as this `negative_loglikelihood`, but I'm getting failures when I make the replacement. I will check it out.